samples/MASMessagingSample/src/main/java/com/ca/mas/masmessagingsample/activity/GroupListActivity.java

```diff
@@ -126,9 +193,120 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 Intent intent = new Intent(this, MessageListActivity.class);
                 startActivity(intent);
                 return true;
+            case R.id.action_logout:
```

Comment:
Same comment regarding the above MASUser variable extraction here.

MASUser user = MASUser.getCurrentUser();
if (user != null && user.isAuthenticated()) {
    user.logout(new MASCallback<Void>() {